Library/Formula/libmemcached.rb

```diff
@@ -15,7 +13,11 @@ class Libmemcached < Formula
   end
 
   option "with-sasl", "Build with sasl support"
+  option "with-static", "Build static library"
+  option "with-memaslap", "Build with load generation and benchmark tool support"
+  option "with-hsieh-hash", "Build with hsieh hashing support"
 
+  depends_on "libevent" # requires --enable-libmemcachedprotocol
```

Comment:
Presumably, it isn't mandatory though, otherwise it wouldn't have built before now? I'm not against adding the dependency, just want to understand how widely it is used elsewhere - What do Debian et al. do on this? Let's kill the comment off either way.